apps/emby/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM docker.io/library/debian:bookworm-slim
+
+ARG VERSION
+
+ENV \
+    DEBIAN_FRONTEND=noninteractive \
+    TZ="Etc/UTC" \
+    UMASK="002" \
+
+WORKDIR /tmp
+
+#hadolint ignore=DL3018,DL3013
+RUN \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        curl \
+        ffmpeg \
+        libsqlite3-0 \
+        libssl3 \
+        libicu72 \
+        fontconfig \
+        libfreetype6 \
+        libass9 \
+    && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN \
+    case "${TARGETPLATFORM}" in \
```

Comment:
Instead of this case statement you can use TARGETARCH directly in the download link